packages/fxa-auth-server/lib/routes/auth-schemes/refresh-token.js

```diff
@@ -33,38 +36,37 @@ module.exports = function schemeRefreshTokenScheme(config, db, oauthdb) {
         const bearerMatchErr = new AppError.invalidRequestParameter(
           'authorization'
         );
-        const refreshToken = bearerMatch && bearerMatch[1];
-        if (refreshToken) {
+        const bearerToken = bearerMatch && bearerMatch[1];
+        if (bearerToken) {
           joi.attempt(bearerMatch[1], validators.refreshToken, bearerMatchErr);
         } else {
           throw bearerMatchErr;
         }
 
-        const refreshTokenInfo = await oauthdb.checkRefreshToken(refreshToken);
-        if (!refreshTokenInfo || !refreshTokenInfo.active) {
+        const tokenId = encrypt.hash(bearerToken);
+        const refreshToken = await oauthDB.getRefreshToken(tokenId);
```

Comment:
here's the replacement for the `oauthdb` call. other changes in this file are to account for changes in the return type